app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -724,6 +761,30 @@ public void onClick(View v) {
                                     }
                                 }
                             });
+
+                    editTextPassword.setOnEditorActionListener(new TextView.OnEditorActionListener() {
```

Comment:
Assign the new EditorListener to a new variable of the same class and just reuse it.

Reply:
@abishekvashok could you please help me with this? I am not able to instantiate OnEditorActionListener. 
Also, it would be easier to instantiate and use the same instance of the OnEditorActionListener if the same action was being carried out inside each listener. But here we are carrying out different actions inside different listeners.

Reply:
@abishekvashok @sauravvishal8797 can I get some guidance with this issue please?

Reply:
Oh ok, can you try to use implement multilevel inheritance and derive a class for each of these click listeners (already present) and editor listeners. It's okay if you want me to merge this but the former approach reduces code. Let me know.

Reply:
@abishekvashok should I create a Utils class where we can write all the functions and then call them where ever required inside the click and the editor listeners?

Reply:
Yep

Reply:
@abishekvashok ok, I'll work on it. Thanks for the guidance:)